.github/workflows/sample-application.yml

```diff
@@ -200,11 +200,20 @@ jobs:
             rn-architecture: 'new'
             ios-use-frameworks: 'no-frameworks'
             build-type: 'production'
+            test-command: 'yarn test-ios-auto' # tests native auto init from JS
```

Comment:
l: due to the test name (`name: Test ${{ matrix.platform }} ${{ matrix.build-type }}` we have a duplicate entry `Test ios production` in the test results.
Wdyt of adding the `test-command` or a new matrix field (e.g. `test-type`) in the name to separate those and easily detect the root of a test failure?